portfolio/src/main/webapp/index.html

```diff
@@ -15,22 +22,37 @@ <h1>Steven's Portfolio</h1>
         <p>Click here to get a random message:</p>
         <button onclick="addRandomGreeting()">Hello</button>
         <div id="greeting-container"></div>
+        <body onload="fetchBlobstoreUrlAndShowForm()">
+            <form action="/data" method="POST">
+                <p>Enter your comment here:</p>
+                <textarea name="text-input">Your text here</textarea>
+                <br/><br/>
+                
+                <p>Number of Comments to Display:</p>
+                <input type="number" name="comments-choice" id="comments-choice" min="1" max="100" value="1">
+                <br/><br/>
 
-        <form action="/data" method="POST">
-            <p>Enter your comment here:</p>
-            <textarea name="text-input">Your text here</textarea>
-            <br/><br/>
-            <p>Number of Comments:</p>
-            <input type="number" name="comments-choice" id="comments-choice" min="1" max="100" value="1">
-            <br/><br/>
-
-            <input type="submit"/>
-        </form>
+                <input type="submit"/>
```

Comment:
Use 2-space indentation for HTML & Javascript as well?

Reply:
Resolved